docs/reference/ImageFont.rst

```diff
@@ -63,6 +63,13 @@ Methods
                  driver prefers; if empty, the renderer may return either
                  mode. Note that the mode is always a string, to simplify
                  C-level implementations.
+    :param direction: Direction of the text.  It can be 'rtl', 'ltr', 'ttb' or 'btt'. Requires  libraqm
+    :param features: A list of font feature to be used during text layout. This is
```

Comment:
This is built on the complex text pr, because I'd been hacking in the test_imagefont stuff, and splitting the patches was going to be hairy and unnecessary.